### PR TITLE
Complete dev & publishing package for the sphinx theme maintenance

### PR DESCRIPTION
**Pull request by rart**

### CODE REVIEW

File: craftercms_sphinx_theme/__init__.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+"""
+Sphinx theme for CrafterCMS docs.
+From https://github.com/rart/craftercms-sphinx-theme.
```
Comment (sumerjabri) [inline craftercms_sphinx_theme/__init__.py]: Please change to new URL

File: README.md [hunk 2]
```diff
@@ -1,2 +1,64 @@
...
+- For production builds:
+  - CSS compression
+  - JavaScript compression
+  - Image compression
+
+## Local Theme Development
+
+### Prerequisites
+
+To use this, you need needs:
```
Comment (sumerjabri) [inline README.md]: remove "needs"

File: gulpfile.babel.js [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+  return gulp.src('src/static/scss/app.scss')
+    .pipe($.sourcemaps.init())
+    .pipe($.sass({
+      includePaths: PATHS.sass
+    })
+      .on('error', $.sass.logError))
+    .pipe($.autoprefixer({
+      browsers: COMPATIBILITY
+    }))
+    // Comment in the pipe below to run UnCSS in production
```
Comment (sumerjabri) [inline gulpfile.babel.js]: Please change the language to become:
// Uncomment the line below to run UnCSS in production

File: package.json [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  "version": "1.0.0",
+  "description": "A sphinx theme for the Crafter CMS docs.",
+  "main": "gulpfile.js",
+  "scripts": {
+    "start": "gulp",
+    "crafter": "gulp build:custom --sphinx --production",
+    "build": "gulp build --production"
+  },
+  "author": "Roy Art <royart@me.com>",
+  "license": "GNU",
```
Comment (sumerjabri) [inline package.json]: Change to GPL 3.0

File: package.json [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    "webpack": "^2.6.1",
+    "webpack-stream": "^3.2.0",
+    "yargs": "^3.8.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/craftercms/craftercms-sphinx-theme.git"
+  },
+  "bugs": {
+    "url": "https://github.com/craftercms/craftercms-sphinx-theme/issues"
```
Comment (sumerjabri) [inline package.json]: Chagne to: https://github.com/craftercms/craftercms/issues

File: setup.py [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+.. _github: https://github.com/craftercms/craftercms-sphinx-theme
+
+"""
+from io import open
+from setuptools import setup
+# from craftercms_sphinx_theme import __version__
+
+setup(
+    name='craftercms_sphinx_theme',
+    description='Crafter CMS Docs theme for Sphinx',
```
Comment (sumerjabri) [inline setup.py]: for Sphinx Doc'

File: setup.py [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+    version='1.1.3a1',
+    url='https://github.com/craftercms/craftercms-sphinx-theme/',
+    license='GNU',
```
Comment (sumerjabri) [inline setup.py]: GPL 3.0

File: setup.py [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+        'sphinx.html_themes': [
+            'craftercms_sphinx_theme = craftercms_sphinx_theme',
+        ]
+    },
+    classifiers=[
+
+        #   3 - Alpha
+        #   4 - Beta
+        #   5 - Production/Stable
+        'Development Status :: 3 - Alpha',
```
Comment (sumerjabri) [inline setup.py]: 5, no?

File: setup.py [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+
+        'License :: OSI Approved :: MIT License',
```
Comment (sumerjabri) [inline setup.py]: GPL 3.0, not MIT

File: src/layouts/default.html [hunk 10]
```diff
@@ -0,0 +1,126 @@
...
+    <nav class="breadcrumbs-container grid-x" aria-label="You are here:" role="navigation">
+      <ul class="breadcrumbs primary">
+        <li><a href="//craftercms.org">Crafter CMS</a></li>
+        <li><a href="/">Docs</a></li>
+        <li><a href="/">3.0</a></li>
+        <li class="disabled">3.0</li>
+        <li><span class="show-for-sr">Current: </span> Cloning</li>
+      </ul>
+      <ul class="breadcrumbs extra">
+        <li><a>Edit on Craphub</a></li>
```
Comment (sumerjabri) [inline src/layouts/default.html]: Craphub? What's this about? Change to GitHub or remove if not related to GitHub.

File: src/pages/index.html [hunk 11]
```diff
@@ -0,0 +1,372 @@
+<div itemprop="articleBody">
```
Comment (sumerjabri) [inline src/pages/index.html]: What's this file for?

File: src/styleguide/index.md [hunk 12]
```diff
@@ -0,0 +1,355 @@
+# The Grid
```
Comment (sumerjabri) [inline src/styleguide/index.md]: What's this file for?

File: src/styleguide/template.html [hunk 13]
```diff
@@ -0,0 +1,91 @@
+<!doctype html>
```
Comment (sumerjabri) [inline src/styleguide/template.html]: What's this file for?

File: yarn.lock [hunk 14]
```diff
@@ -0,0 +1,6741 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (sumerjabri) [inline yarn.lock]: Is this an output file? If so, remove.

Comment (rart) [reply yarn.lock]: It is, but it needs to be committed. It's what controls and ensures correct dependencies together with the package.json.

File: src/pages/index.html (hunk 11 above)
Comment (rart) [reply src/pages/index.html]: For development. A sample of the content to be styled.

**Comment by rart** [comment]
Updated.

**Pull request merged**